VSDiagnostics/VSDiagnostics/VSDiagnostics/VSDiagnosticsResources.resx

```diff
@@ -421,23 +421,14 @@
   <data name="EnumCanHaveFlagsAttributeCodeFixTitle" xml:space="preserve">
     <value>Add Flags attribute</value>
   </data>
-  <data name="ConditionIsAlwaysTrueAnalyzerMessage" xml:space="preserve">
-    <value>Condition is always true.</value>
+  <data name="ConditionIsConstantAnalyzerMessage" xml:space="preserve">
+    <value>Condition is constant.</value>
```

Comment:
I would suggest that the message becomes "The condition is always {0}" and as argument you pass in `true`/`false`